op-node/rollup/derive/span_batch_txs.go

```diff
@@ -468,3 +468,40 @@ func (sbtx *spanBatchTxs) AddTxs(txs [][]byte, chainID *big.Int) error {
 	sbtx.totalBlockTxCount += totalBlockTxCount
 	return nil
 }
+
+func (sbtx *spanBatchTxs) revertLastBatch(txs []hexutil.Bytes) {
```

Comment:
`spanBatchTxs` don't have a native concept of a "Batch", so it doesn't make sense to call this `revertLastBatch`, especially when the parameter is `txs`. Suggest to name this `revertTxs`

Reply:
I renamed it to `revertLastTxs` since it can only revert the `last` txs.